lib/mumukit/content_type/md_emoji.rb

```diff
@@ -20,102 +20,114 @@
+    pencil pencil2 penguin pensive performing_arts persevere person_frowning person_with_ball person_with_blond_hair person_with_pouting_face phone
+    pick pig pig2 pig_nose pill pineapple pisces pizza place_of_worship plus1 point_down point_left point_right point_up point_up_2 police_car poodle
+    poop popcorn post_office postal_horn postbox potable_water pouch poultry_leg pound pouting_cat pray prayer_beads princess printer punch
+    purple_heart purse pushpin put_litter_in_its_place question rabbit rabbit2 racehorse racing_car racing_motorcycle radio radio_button
+    radioactive_sign rage rage1 rage2 rage3 rage4 railway_car railway_track rain_cloud rainbow raised_hand raised_hand_with_fingers_splayed
+    raised_hands raising_hand ram ramen rat recycle red_car red_circle registered relaxed relieved reminder_ribbon repeat repeat_one restroom
+    reversed_hand_with_middle_finger_extended revolving_hearts rewind ribbon rice rice_ball rice_cracker rice_scene right_anger_bubble ring robot_face
+    rocket rolled_up_newspaper roller_coaster rooster rose rosette rotating_light round_pushpin rowboat ru rugby_football runner running
+    running_shirt_with_sash sa sagittarius sailboat sake sandal santa satellite satisfied saxophone scales school school_satchel scissors scorpion
+    scorpius scream scream_cat scroll seat secret see_no_evil seedling seven shamrock shaved_ice sheep shell shield shinto_shrine ship shipit shirt
+    shit shoe shopping_bags shower showman sign_of_the_horns signal_strength six six_pointed_star ski skier skull skull_and_crossbones sleeping
+    sleeping_accommodation sleepy sleuth_or_spy slightly_frowning_face slightly_smiling_face slot_machine small_airplane small_blue_diamond
+    small_orange_diamond small_red_triangle small_red_triangle_down smile smile_cat smiley smiley_cat smiling_imp smirk smirk_cat smoking snail
+    snake snow_capped_mountain snow_cloud snowboarder snowflake snowman sob soccer soon sos sound space_invader spades spaghetti sparkle sparkler
+    sparkles sparkling_heart speak_no_evil speaker speaking_head_in_silhouette speech_balloon speedboat spider spider_web spiral_calendar_pad
+    spiral_note_pad spock-hand sports_medal squirrel stadium star star2 star_and_crescent star_of_david stars station statue_of_liberty
+    steam_locomotive stew stopwatch straight_ruler strawberry stuck_out_tongue stuck_out_tongue_closed_eyes stuck_out_tongue_winking_eye
+    studio_microphone sun_behind_cloud sun_behind_rain_cloud sun_small_cloud sun_with_face sunflower sunglasses sunny sunrise sunrise_over_mountains
+    surfer sushi suspect suspension_railway sweat sweat_drops sweat_smile sweet_potato swimmer symbols synagogue syringe table_tennis_paddle_and_ball
+    taco tada tanabata_tree tangerine taurus taxi tea telephone telephone_receiver telescope tennis tent the_horns thermometer thinking_face
```

Comment:
This is weird: `thinking_face` is not actually rendered by github (:thinking_face:), but :thinking: is. 

Moreover, `thinking` is not present in this set. 

This is just an example, there may be more inconsistencies.